doc/user_guide/user_guide.md

```diff
@@ -1066,3 +1066,53 @@ changes.
 [delta-io]: https://delta.io/
 [delta-storage]: https://docs.delta.io/latest/delta-storage.html
 [delta-history]: https://docs.delta.io/latest/delta-utility.html#history
+
+## Hadoop Distributed Filesystem (HDFS)
+
+The [Hadoop distributed file system (HDFS)]() is a distributed, scalable, and
+portable file system written in Java for the Hadoop framework
+
+When the Hadoop datanodes and Exasol cluster are installed in the same (virtual)
```

Comment:
I asked myself: How is it if they are not in the same network?

Reply:
Did you just assume it for the example or is it not possible at all?

Reply:
Yes, it is not possible. Added a notice.